### PR TITLE
Add debugger instructions

### PR DESCRIPTION
**Pull request by rinatkhaziev**
## Description

I got frustrated enough to actually do this. It's great.
Screenshot 2022-12-09 at 4 20 16 PM

Screenshot 2022-12-09 at 4 20 35 PM

## Steps to Test

Check out the PR, follow the instructions and make sure it works!

### CODE REVIEW

**Comment by rinatkhaziev** [comment]
Props @chriszarate for giving me tips along the journey.

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -29,6 +29,35 @@ VIP_PROXY="" API_HOST=http://localhost:4000 node ./dist/bin/vip app
...
 ```
 
+### Using debugger
+
+Who doesn't like a good console.log for debugging?
+Well, sometimes it's insufficient, luckily it's not too complicated to use a debugger.
+
+1. First, make sure to run the `npm run build:watch`, this will generate source maps
+1. (Optional) run `npm link`
+1. Run the command you want via `node --inspect`, like so:  `node --inspect /opt/homebrew/lib/node_modules/@automattic/vip/dist/bin/vip-dev-env-import-sql.js`
```
Comment (chriszarate) [inline CONTRIBUTING.md]: confusing to say `npm link` is optional but then target the linked path. i would document it using the local path. `npm link` seems like a totally separate decision that has no effect on debugging

```suggestion
1. Run the command you want via `node --inspect`, like so:  `node --inspect ./dist/bin/vip-dev-env-import-sql.js`
```

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -29,6 +29,35 @@ VIP_PROXY="" API_HOST=http://localhost:4000 node ./dist/bin/vip app
...
+1. Note the port the debugger is listening on:
+```
+Debugger listening on ws://127.0.0.1:9229/db6c03e9-2585-4a08-a1c6-1fee0295c9ff
+For help, see: https://nodejs.org/en/docs/inspector
+```
+5. In your editor of choice attach to the debugger.
+6. For VSCode: Hit 'Run and Debug' panel, hit the "gear" icon (open launch.json), make your `Attach` configuration entry to look like so:
```
Comment (chriszarate) [inline CONTRIBUTING.md]:
```suggestion
5. In your editor of choice attach to the debugger. For VSCode: Hit 'Run and Debug' panel, hit the "gear" icon (open launch.json), make your `Attach` configuration entry to look like so:
```

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -29,6 +29,35 @@ VIP_PROXY="" API_HOST=http://localhost:4000 node ./dist/bin/vip app
...
+Make sure the the `port` and `runtimeExecutable` match
```
Comment (chriszarate) [inline CONTRIBUTING.md]: Does this elaboration make sense?

```suggestion
Make sure the `port` matches the port from step 3, and the `runtimeExecutable` matches the exact `node` executable you ran. If you use a version manager like `nvm`, its especially important to check this.
```

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -29,6 +29,35 @@ VIP_PROXY="" API_HOST=http://localhost:4000 node ./dist/bin/vip app
...
+		"name": "Attach",
+		"port": 9229,
+		"request": "attach",
+		"skipFiles": ["<node_internals>/**"],
+		"type": "node",
+		"runtimeExecutable": "/Users/rinat/.nvm/versions/node/v14.18.2/bin/node"
+	}
+```
+7. Set your breakpoints and whatnot, hit the play button.
+8. Squash them bugs.
```
Comment (chriszarate) [inline CONTRIBUTING.md]: 🐛🔨

**Review by chriszarate — APPROVED** [review]
amazing!!

**Pull request merged**